libs/orbit-essentials/src/http.rs

```diff
@@ -61,3 +61,37 @@ fn cbor_encode(value: &impl Serialize) -> Vec<u8> {
 pub fn certified_data_for_skip_certification() -> [u8; 32] {
     skip_certification_asset_tree().digest()
 }
+
+pub fn not_found() -> HttpResponse {
+    HttpResponse {
+        status_code: 404,
+        headers: vec![HeaderField("Content-Type".into(), "text/plain".into())],
+        body: "404 Not Found".as_bytes().to_owned(),
+    }
+}
+
+pub fn parse_path(url: &str) -> Option<&str> {
+    url.split('?').next()
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_parse_path() {
+        assert_eq!(parse_path("/long/path?param"), Some("/long/path"));
+        assert_eq!(parse_path("/path?query=1"), Some("/path"));
+        assert_eq!(parse_path("/path#unwanted"), Some("/path#unwanted"));
+        assert_eq!(parse_path("/"), Some("/"));
+        assert_eq!(parse_path(""), Some(""));
+    }
+
+    #[test]
+    fn test_cbor_encode() {
```

Comment:
what is this test covering?

Reply:
It's a dummy test for the `cbor_encode` function from this module, which had no test. I was mostly curious about the output, so I left it there. I can remove it if it's useless.

Reply:
i see, let's keep then, but maybe remove the `println!`